app/src/main/java/org/kinecosystem/kinit/viewmodel/restore/RestoreWalletViewModel.kt

```diff
@@ -43,7 +43,12 @@ class RestoreWalletViewModel {
 
     fun onAnswersSubmit() {
         answersSubmitted.set(true)
-        val restoredAccount = walletService.importBakedUpAccount(qrCode, passphrase)
+        passphrase = answers.joinToString(separator = "")
+        var restoredAccount = walletService.importBakedUpAccount(qrCode, passphrase)
+        if (restoredAccount == null || restoredAccount.publicAddress == null) {
+            passphrase = answers.joinToString(separator = "") { it.trim() }
+            restoredAccount = walletService.importBakedUpAccount(qrCode, passphrase)
```

Comment:
cool - just think we should first check for the trimmed version as it will have more chances to work